app/frontend/src/pages/Profile/ProfilePage.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import * as images from "../../pages/Auth/images";
+import Sidebar from '../../components/Sidebar'
+import LikedPosts from "./LikedPosts";
+
+const ProfilePage = () => {
+
+    const user = {
+        name: 'Mahmut Demir',
+        title: 'CEO',
+        company: 'Peak Games',
+        following: 144,
+        followers: 123,
+        about: 'Greetings fellow gamers! As an avid player navigating the vast landscapes of virtual worlds, I find joy in the pixels and excitement in the ever-evolving narratives of video games. From intense battles in fantasy realms to strategic maneuvers in competitive arenas, gaming has become my digital sanctuary.',
+    };
+
+    const postData = [
+        {
+            header: 'Resident Evil 4 Remake',
+            content: "Looking for someone to join me in my Resident Evil 4 adventure! Let's team up and face the horrors together.️ #ResidentEvil4 #GamingBuddyWanted",
+            date: '27.10.2023  00.00',
+        },
+        {
+            header: 'Fifa',
+            content: "FIFA is one of the most popular football simulation games developed by EA Sports. It offers an immersive gaming experience with realistic graphics, player mechanics, and stadiums.",
+            date: '28.10.2023  02.02',
+        },
+        {
+            header: 'The Witcher 3: Wild Hunt',
+            content: "The Witcher 3 is an unforgettable gaming experience. Its open world, rich storytelling, and captivating characters make it a must-play RPG. If you love epic adventures, this one's a masterpiece.",
+            date: '29.10.2023  06.42',
+        },
+    ];
+
+    return (
+        <div className='flex flex-row bg-[#FFF4E0]'>
+            <div className='w-1/5 flex flex-col h-screen'>
+                <Sidebar/>
+            </div>
+            <div className='w-4/5 flex flex-col p-5 justify-center items-center'>
+                <div className="compact text-[#FFF4E0] bg-[#4D4D4D]  text-sky-800 shadow-xl  p-4 flex w-4/5 h-1/4 mb-2 rounded-xl">
```

Comment:
Since you gave text color twice, it shows texts in wrong color. You should remove the "text-sky-800" part to make texts readable.